internal/exec/stages/files/filesystemEntries.go

```diff
@@ -56,7 +56,8 @@ func (s *stage) createCrypttabEntries(config types.Config) error {
 	}
 	extrafiles := []filesystemEntry{}
 	for _, luks := range config.Storage.Luks {
-		out, err := exec.Command(distro.CryptsetupCmd(), "luksUUID", *luks.Device).CombinedOutput()
+		devAlias, err := util.CreateDeviceAlias(*luks.Device)
```

Comment:
Hmm, is it actually worth creating a device alias here since we're just doing a single operation on the LUKS device anyway?

Reply:
Seems like a general best practice in Ignition right?

Reply:
I think it'd be more correct here to just reuse the same devalias that `disks` created. That way we're sure we're operating on the same LUKS device. IOW

```suggestion
		devAlias := util.DeviceAlias(*luks.Device) 
```

?

Reply:
Yup, should just be re-using the existing alias.